plugins/localization/proguard-rules.pro

```diff
@@ -0,0 +1,25 @@
+# Add project specific ProGuard rules here.
```

Comment:
Doesn't look like this ProGuard files needed.

Reply:
Removed